lenovo/legion/15ich/default.nix

```diff
@@ -2,7 +2,7 @@
 
 {
   imports = [
-    ../../../common/cpu/intel
+    ../../../common/cpu/coffee-lake
```

Comment:
error: path '/nix/store/ngfdd9j2fvz3zf5a6vc8zvcg5yxrllql-source/common/cpu/coffee-lake' does not exist

Reply:
Huh, I have been running it forever, but it was way behind master until today. Will double check.